libs/platform/src/lib/components/form/radio-group/radio-group.component.spec.ts

```diff
@@ -151,7 +151,7 @@ class TestComponentList {
     radioButtons: RadioButtonComponent;
 }
 
-describe('GroupRadioButtonComponent', () => {
+xdescribe('GroupRadioButtonComponent', () => {
```

Comment:
Is there a reason for excluding this test suite with `xdescribe`?

Reply:
This test case behaving flacky. it is failing in local some times.

Reply:
Does this "flaky" test suite block the PR from being merged? Since this file is not part of Checkbox Component, it's a little odd having this code change in the PR.

Commenting out, or rather skipping a unit test suite which will be merged into master is bad practice. We should probably create an issue and try and fix the unit test(s) in another PR.

As for now, if this test suite is blocking this PR from being "mergeable" then I guess it's OK to skip it, but definitely create an issue so we don't forget it. If it's not blocking this PR then I would revert it back.

Reply:
have removed these unwanted changes.